test-it/karma/fixtures/javascriptinvoke/ObjectType.via

```diff
@@ -0,0 +1,50 @@
+define(MyVI dv(VirtualInstrument (
+    Locals: c(
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) error1)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) error2)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) error3)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) error4)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) error5)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) error6)
+        e(c(e(.Boolean status) e(.Int32 code) e(.String source)) error7)
+        e(.JavaScriptRefNum ref1)
+        e(.JavaScriptRefNum ref2)
+        e(.JavaScriptRefNum ref3)
+        e(.JavaScriptRefNum ref4)
+        e(.JavaScriptRefNum ref5)
+        e(dv(.String "Foo") name1)
+        e(dv(.String "FooBar") name2)
+        e(dv(.Int32 0) length1)
+        e(dv(.Int32 0) length2)
+        e(dv(.Boolean false) isEqual)
+        e(dv(.Boolean false) isNotEqual)
+        e(dv(.Boolean false) isNotANumPathRefnum1)
+        e(dv(.Boolean false) isNotANumPathRefnum2)
+        e(dv(.Boolean false) isSharedRef)
+        e(dv(.Boolean false) isSharedPrimRef)
+        e(.Occurrence occurrence)
+    )
+    clump (
+        // Get js object ref
+        JavaScriptInvoke(occurrence false error1 'NI_GetObjectFunction' ref1 name1)
```

Comment:
Talked to Gandi about a couple of more test cases including checking for "not-a-refnum" on a ref before calling JSInvoke on that.